account/models.py

```diff
@@ -1,3 +1,15 @@
 from django.db import models
 
-# Create your models here.
+class Account(models.Model):
+    name           =   models.CharField(max_length=50)
+    email          =   models.EmailField(max_length=50)
+    password       =   models.CharField(max_length=100)
+    language       =   models.CharField(max_length=50, null=True)
+    country        =   models.CharField(max_length=50, null=True)
+    birthday       =   models.IntegerField(default=0,  null=True)
```

Comment:
```suggestion
    birthday       =   models.DateField(null=True)
```
생년월일은 날짜 이므로 datefield로 관리하시는 것이 좋습니다 :)